### DISCUSSION TITLE
Different machines gave different results despite running the same code

### DISCUSSION THREAD

**Discussion by daminho** [ANLAB-KAIST/KENSv3#104] [Project Q&A 1. Application Layer (Socket)]
Hi, I and my teammate faced the problem that we ran the same code, it showed the test results on my computer but gave an error on her computer. The error is "Assertion `nsIter != appIter->second.fdToDomain.end()' failed (E_Host.cpp : 107).

**Comment by whkim0** [comment]
A bug in your code can cause that error. Also, it could be a memory bug. Perhaps one of your team members is using macOS, right?
This will be helpful: https://github.com/ANLAB-KAIST/KENSv3/wiki/Tip:-Debugging-Memory-Bugs-(Only-for-macOS-and-Linux)
